lib/search.js

```diff
@@ -80,7 +80,7 @@ module.exports = (elastic, queryParams, next) => {
           filter: {
             exists: {field: 'locations'}
           },
-          weight: 1.2
+          weight: 10
```

Comment:
Yeah, location > image, but most of the objects that have images are on display anyway, so they end up doubly boosted.